lib/benchee/formatters/console.ex

```diff
@@ -62,27 +62,52 @@ defmodule Benchee.Formatters.Console do
     max_label_width + 1
   end
 
-  defp job_reports(jobs, label_width) do
-    Enum.map(jobs, fn(job) -> format_job job, label_width end)
+  defp job_reports(jobs, units, label_width) do
+    Enum.map(jobs, fn(job) -> format_job job, units, label_width end)
+  end
+
+  defp units(jobs) do
+    # Produces a map like
+    #   %{run_time: [12345, 15431, 13222], ips: [1, 2, 3]}
+    collected_values = jobs
+    |> Enum.flat_map(fn({_name, job}) -> Map.to_list(job) end)
+    # TODO: Simplify when dropping support for 1.2
+    # For compatibility with Elixir 1.2. In 1.3, the following group-reduce-map
+    # can b replaced by a single call to `group_by/3`
+    #   Enum.group_by(fn({measurement, _}) -> measurement end, fn({_, value}) -> value end)
+    |> Enum.group_by(fn({measurement, _unit}) -> measurement end)
```

Comment:
is the second parameter here really the unit? Isn't it the value? e.g. shouldn't this be something like `{"average", 45454}`.

If I'm correct, then measurement also seems not totally spot on to me. Maybe "statistic"/"statistic_name"/"stat" - but apprently `measurement` is exactly the correct word. Weird I always thought of `measurement` as something that I measure directly e.g. a run time and not a derived computed value but Wikipedia calls it like this:

>  The median is a commonly used measure of the properties of a data set in statistics and probability theory.